pyro/distributions/lkj.py

```diff
@@ -62,65 +60,55 @@ class _PartialCorrToCorrCholeskyTransform(Transform):
 
     Reference:
 
-    [1] `Generating random correlation matrices based on vines and extended onion method`,
-    Daniel Lewandowski, Dorota Kurowicka, Harry Joe
+    [1] `Cholesky Factors of Correlation Matrices`,
+    Stan Reference Manual v2.18, Section 10.12
```

Comment:
Lewandowski does not mention about this transform (only transform from PartialCorr to Corr) so Stan Reference Manual will be our main reference.